cmd/lakectl/cmd/doctor.go

```diff
@@ -0,0 +1,95 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+	"net/url"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/treeverse/lakefs/pkg/api"
+)
+
+var (
+	ErrCredential       = errors.New("credential error")
+	ErrWrongEndpointURI = errors.New("wrong endpoint_uri error")
+	ErrUnknownConfig    = errors.New("unknown configuratioin error")
```

Comment:
```suggestion
	ErrCredential       = errors.New("bad credential")
	ErrWrongEndpointURI = errors.New("wrong endpoint_uri")
	ErrUnknownConfig    = errors.New("unknown configuration")
```

Errors should not include the word "error" in their message.  E.g. UN*X says "file not found" not "file not found error".  Also fixed some spelling.